### ISSUE TITLE
Website still says that the version 8.2

### ISSUE THREAD

**Issue by yurivict** [RTimothyEdwards/magic#76]
http://opencircuitdesign.com/magic/

**Comment by RTimothyEdwards** [comment]
Yes, I know. . . it's just rather low on my priority list.  I guess when somebody bugs me about it, it gets bumped up on the list, so now that you've pointed it out I'm going to have to go do something about it. . .
